src/Pages/HugeAss/HugeAss.js

```diff
@@ -1,9 +1,105 @@
 import React from "react";
+import "./HugeAss.css";
+import Patreon from "../../Components/Patreon/Patreon";
+import hugeassPic from "../../images/hugeass.jpg";
+import podIcon from "../../images/podIcon.webp";
+import discordIcon from "../../images/discordIcon.png";
+import youtubeIcon from "../../images/youtubeIcon.png";
 
 export default function HugeAss() {
     return (
-        <>
-            <p>Huge Ass</p>
-        </>
+        <div className="outerContainer">
+            <img
+                className="projectHeader"
+                src={hugeassPic}
+                alt="beerGoggles logo"
+            />
+            <div className="hugeAssContainer">
+                <div className="hugeAssIntroContainer">
+                    <div className="projectDescription">
+                        <p>
+                            HUGE ASS was spawned from the covid-19 pandemic. The
+                            HUGE Crew wanted to make something together but also
+                            remote, so we decided to do some asynchronous speed
+                            running. Each player spends 30 mins playing a
+                            particular game and records it. We then sync up the
+                            footage and do some fun commentary over top and see
+                            who was faster. Eventually, finding out who the
+                            HUGEst ASS truely is.
+                        </p>
+
+                        <p>
+                            You can check out Season One on the HUGE Youtube
+                            Channel. Season Two is availble by {""}
```

Comment:
not sure what this little `{""}` is for

Reply:
without it there isn't a space between "by" and "subscribe"